setup.py

```diff
@@ -38,8 +38,8 @@ def load_deps(file_name):
 install_requires = load_deps("requirements.txt")
 tests_require = load_deps("dev-requirements.txt")
 extras_require = {
-    "s3": ["s3fs~=0.5"],
-    "az": ["adlfs~=0.5"],
+    "s3": ["s3fs==2023.9.2"],
+    "az": ["adlfs==2023.9.0"],
```

Comment:
Pretty sure you have a separate PR for this.

Reply:
#481

Reply:
Your'e right reverted.